### PR TITLE
Update docs to reflect new bot status.

### PR DESCRIPTION
**Pull request by ehsannas**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -87,6 +87,9 @@ usual things:
 *   Identify potential functional problems.
 *   Identify code duplication.
 *   Ensure the unit tests have enough coverage.
+*   Ensure continuous integration (CI) bots run on the PR. If not run (in the
+    case of PRs by external contributors), add the "kokoro:run" label to the
```
Comment (dneto0) [inline CONTRIBUTING.md]: I thought only project members can add the label?  I think this needs to be rephrased to *request* a project maintainer to add the label.

Comment (ehsannas) [reply CONTRIBUTING.md]: You're right. But I have added these new lines in the section titled
#### For maintainers: Reviewing a PR

Comment (dneto0) [reply CONTRIBUTING.md]: Bah.  My bad. I didn't see the context.

**Comment by dneto0** [comment]
Rebased and pushed into master as  c02d216648

**Pull request closed without merging**